core/storage/classifier/gae_models_test.py

```diff
@@ -52,3 +53,40 @@ def test_create_new_classifier_runs_successfully(self):
         self.assertEqual(classifier.algorithm_id, 'LDAStringClassifier')
         self.assertEqual(classifier.classifier_data, {'alpha': 1.0})
         self.assertEqual(classifier.data_schema_version, 1)
+
+
+class TrainClassifierJobModelUnitTests(test_utils.GenericTestBase):
```

Comment:
Ditto. We might need to rename this to ClassifierTrainingJobModelUnitTests as well.